projects/laji/src/app/shared/service/form-permission.service.ts

```diff
@@ -102,46 +102,46 @@ export class FormPermissionService {
   }
 
   getRights(form: Form.List): Observable<Rights> {
-    const notLoggedIn = {
+    const {collectionID} = form;
+    const notLoggedIn$ = this.getFormPermission(collectionID).pipe(map(permissions => ({
```

Comment:
Renamed as `notLoggedInRights$`.

Seems to me that rights/and permissions are not interchangeable. They have different structure, "rights" using the interface `Rights`, which is an internal model in laji.fi code, and "permissions" using `FormPermission` whose structure comes from the API.

Reply:
I ended up with `notLoggedInRights$` because:

1. I think the helps readability to have "not logged in" in the beginning. This is subjective though.
2. I dropped the word "while", because it feels redundant to me. Using fewer words helps readability. Also subjective...

But adding "rights" was a good idea anyways.